lib/web_catalog/api_extractor.es6.js

```diff
@@ -413,11 +414,22 @@ foam.CLASS({
         //
 
         let apiCatalog = {};
+        let fnNameMap = {};
```

Comment:
Probably better to use a [JS Map (built-in)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)

`hasOwnProperty` => `has`